CardSharp.Tests/ShufflerTest.cs

```diff
@@ -8,12 +8,13 @@ public class ShufflerTest {
         public void FisherYates_GivenKnownSequence_ShufflesIntoSequence() {
             // This test really just exercises the algorithm
             var randomizer = new FakeRandomizer(2, 0);
-            var deck = new Deck<int>(new[] { 3, 2, 1 });
+            var deck = new Deck<int>(new Card<int>[] { 3, 2, 1 });
             var shuffler = new Shuffler(randomizer);
 
-            var shuffledDeck = shuffler.FisherYates(deck);
+            shuffler.FisherYates(deck);
 
-            CollectionAssert.AreEqual(new[] { 1, 2, 3 }, shuffledDeck);
+            var expected = new Deck<int>(new Card<int>[] { 1, 2, 3 });
```

Comment:
So, long form of what's happening here if you didn't have the implicit operators is:
```c#
new Deck<int>(new Card<int>[] { new Card<int>(1), new Card<int>(2), new Card<int>(3) });
```
Is that right?

If `Deck<T>` is a `List<Card<T>>`, then you could probably just have a deck constructor like:
```c#
public Deck(params T[] items)
{
    /* whatever */
}
```
Then usage becomes
```c#
var expected = new Deck<int>(new[] { 1, 2, 3 });
```

Then the consumer doesn't need to worry about cards, they just operate based on `<T>`

Reply:
Oh, after writing that, I noticed that is verbatim what you used to have. I think its cleaner for a consumer to only operate on their data type, and you can hide the details in the class internals.

Based on your `Card.Make` idea, it might be nice to have a `Deck<T> Deck.Create<T>(params T[] items)` static method. Then you just do `var deck = Deck.Create(new[] { 1, 2, 3});`

Reply:
> So, long form of what's happening here if you didn't have the implicit operators is:

Yep!

> Based on your Card.Make idea, it might be nice to have a Deck<T> Deck.Create<T>(params T[] items) static method. Then you just do var deck = Deck.Create(new[] { 1, 2, 3});

I agree, and I ended up implementing the object initializer pattern to make it even tighter - you can do `new Deck<int>{ 1, 2, 3 }`